### PR TITLE
use noCount

### PR DESCRIPTION
**Pull request by SimonSchick**

### CODE REVIEW

**Comment by SimonSchick** [comment]
[Experiment with removing the x-total-count header for performance](https://trello.com/c/CrEH0rRx/103-experiment-with-removing-the-x-total-count-header-for-performance)

**Comment by SimonSchick** [comment]
@jackcook pls

File: Pod/Classes/Routes/ChannelsRoutes.swift [hunk 1]
```diff
@@ -183,7 +183,12 @@ public class ChannelsRoutes {
      :param: completion An optional completion block with the retrieved channels' data.
      */
     public func getChannels(_ requestType: ChannelsRequestType = .all, page: Int = 0, completion: ((_ channels: [BeamChannel]?, _ error: BeamRequestError?) -> Void)?) {
-        var params = ["order": "viewersCurrent:desc", "where": "suspended.eq.0,online.eq.1", "page": "\(page)"]
+        var params = [
+            "order": "viewersCurrent:desc",
+            "where": "suspended.eq.0,online.eq.1",
+            "page": "\(page)",
+            "noCount": "1",
```
Comment (jackcook) [inline Pod/Classes/Routes/ChannelsRoutes.swift:190]: It's supposed to be a string, not an int?

**Comment by jackcook** [comment]
haha sorry I forgot about this PR

File: Pod/Classes/Routes/ChannelsRoutes.swift (hunk 1 above)
Comment (jackcook) [reply Pod/Classes/Routes/ChannelsRoutes.swift:190]: actually nevermind, you're correct

**Review by jackcook — APPROVED** [review]

**Pull request merged**